### DISCUSSION TITLE
Low Latency setting question

### DISCUSSION THREAD

**Discussion by mlash31887** [Mygod/VPNHotspot#227] [Q&A]
I sometimes use my phone as a hotspot for my xbox. Would the low latency setting improve my connection on my xbox or does the low latency setting only help when you are using an outside wifi connection to get internet for your phone.  Thanks for any clarification.

**Comment by Mygod** [comment]
This mode is a hint to the system that the Wi-Fi should be kept as low latency mode. What it does actually will depend on your device and ROM.

**Reply by mlash31887** [reply]
Thanks. Much appreciated.

**Reply by Mygod** [reply]
You can read more about it here: https://source.android.com/devices/tech/connect/wifi-low-latency

**Reply by mlash31887** [reply]
Exactly the info I was looking for.  Thanks again.
